source/geod24/LocalRest.d

```diff
@@ -475,6 +509,51 @@ public final class RemoteAPI (API) : API
         {
             C.send(this.childTid, TimeCommand(d, dropMessages));
         }
+
+        /***********************************************************************
+
+            Filter any requests issued to the provided method.
+            Calling the API endpoint will throw an exception,
+            therefore the request will fail.
+
+            Use via:
+
+            -----
+            interface API { void call(); }
+            class C : API { void call() { } }
+            auto obj = new RemoteAPI!API(...);
+            obj.filter!(obj.call);
```

Comment:
> Why not just pass API.method here ?

That works too.

> Also could you add an example with overloads ?

Doesn't support it yet. It would be hard to support this on the call side without making it awkward though. Do you have any ideas?

Normally in D to match a specific overload you'd have to do something like `void function(int, float) foo = &object.foo`, but that's ugly to use.